examples/image_resources/image_resources.bat

```diff
@@ -0,0 +1,51 @@
+:: Copyright (c) 2020 Nekokatt
+::
+:: Permission is hereby granted, free of charge, to any person obtaining a copy
+:: of this software and associated documentation files (the "Software"), to deal
+:: in the Software without restriction, including without limitation the rights
+:: to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+:: copies of the Software, and to permit persons to whom the Software is
+:: furnished to do so, subject to the following conditions:
+::
+:: The above copyright notice and this permission notice shall be included in all
+:: copies or substantial portions of the Software.
+::
+:: THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+:: IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+:: FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+:: AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+:: LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+:: OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+:: SOFTWARE.
+
+:: Script to set the PYTHONPATH variable up to point to the root of this repository.
+
+@echo off
+
+if "%~1"=="" (
+    echo Please pass a bot token as the first parameter to this script!
+    :: This is used so that if the script is executed by double clicking it,
```

Comment:
These in-statement comments are being interpreted as a drive letter on my system in both of these, leading to it sending out a unnecessary warning before exiting with status 1
```
>image_resources.bat
Please pass a bot token as the first parameter to this script!
The system cannot find the drive specified.
Press any key to continue . . .
```

Reply:
comments in batch are `REM blah blah`

Reply:
Hmm interesting.... This is weird because there are a lot of comments before this and those didnt spam anything. Further more, that is the way to mark as comments in batch (as I learned today).

Could you try removing those lines (that one and the following one) and see if it still errors?

Reply:
replacing "::" with "REM" on those two lines in both of the bat files got rid of the warning.

Reply:
> comments in batch are REM blah blah

Most of resources I found said that both `rem` and `:: ` were fine and I went with `:: ` as it made it more readable

Reply:
> replacing "::" with "REM" on those two lines in both of the bat files got rid of the warning.

Ig I will change all the comments then

Reply:
might be the indent, im not sure. Batch is archaic.

Reply:
Well, should be fixed :+1: @FasterSpeeding mind trying again?